src/interfaces/studyManage.d.ts

```diff
@@ -25,6 +25,6 @@ export interface studyManageType {
   numberOfMembers: number;
   numberOfRecruits: number;
   studyMembers: studyManageMemberType[];
-  numberOfApplicant: number;
+  numberOfApplicants: number;
   studyApplicants: studyManagestudyApplicantsType[];
```

Comment:
```suggestion
  studyApplicants: studyManageStudyApplicantsType[];
```
위쪽에 studyManageStudyApplicantsType 인터페이스를 쓰는 것이라면 오탈자가 있습니다!